src/common/utility.ts

```diff
@@ -438,19 +437,6 @@ const getSubdomain = () => isBrowser() && window.location.hostname.split('.')[0]
 export const isEuDomain = () =>
     !!eu_domains.some((eu_sub_domain) => eu_sub_domain.test(getSubdomain()))
 
-export const isUkDomain = () =>
-    !!uk_domains.some((uk_sub_domain) => uk_sub_domain.test(getSubdomain()))
-
-export const handleRedirect = (residence: string, current_client_country: string): boolean => {
```

Comment:
Don't we need `handleRedirect` at all?

Reply:
Yes I assume I deleted mistakenly , I revert it back thanks 🙏